pyproject.toml

```
@@ -339,6 +340,9 @@ manylinux-s390x-image = "manylinux_2_28"
 test-skip = [
   "cp3*t-*", # no freethreading qiskit wheels
   "cp*-win_arm64", # no numpy, qiskit, ... wheels
+  # TODO(denialhaag): Remove when all dependencies support Python 3.14
+  # https://github.com/munich-quantum-toolkit/qmap/issues/718
```

```suggestion
  #   https://github.com/munich-quantum-toolkit/qmap/issues/718
```
mainly for proper highlighting of the todo in IDEs